bindings/python/src/OpenSpaceToolkitMathematicsPy/Objects/Interval.cpp

```diff
@@ -39,6 +39,8 @@ inline void                     OpenSpaceToolkitMathematicsPy_Objects_Interval (
         .def("intersects", &Interval<Real>::intersects)
         .def("contains_real", +[] (const Interval<Real>& anInterval, const Real& aReal) -> bool { return anInterval.contains(aReal) ; })
         .def("contains_interval", +[] (const Interval<Real>& anInterval, const Interval<Real>& anOtherInterval) -> bool { return anInterval.contains(anOtherInterval) ; })
+        .def("get_intersection_with", &Interval<Real>::getIntersectionWith)
```

Comment:
Let's maybe add the `arg` to each function of this file while we are at it, what do you think?

Reply:
Great call, the python bindings definitely need better help text 👍